### PR TITLE
Reset filter after click on Logo or Swift Evolution title

### PR DESCRIPTION
**Pull request by uxitten**
Imagine we choice some filter and after that, we expect reset all filter after a click on `Logo` or `Swift Evolution` title

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -31,8 +31,8 @@
         <div class="app">
             <header>
                 <span class="header-contents">
-                    <a id="logo" href="#" title="Swift Evolution" role="img" aria-label="Swift Evolution"></a>
-                    <h1 id="title"><a href="#">Swift Evolution</a></h1>
+                    <a id="logo" href="." title="Swift Evolution" role="img" aria-label="Swift Evolution"></a>
```
Comment (krilnon) [inline index.html:34]: Although from a coding perspective it’s expedient to do this, does it really make sense to reload the entire page just to toggle off the filter?

Seems like we should add a function for this, roughly: 

```javascript
if(!document.querySelector('.filter-toggle').classList.contains('hidden')) { toggleFiltering() }
```

**Comment by benrimmington** [comment]
@krilnon Linking to the repository was [my suggestion](https://github.com/apple/swift-evolution/pull/592#issuecomment-274271980) from a few years ago, but I agree with [your reply](https://github.com/apple/swift-evolution/pull/592#issuecomment-274634696) that it would be too much of a context switch.

I think clicking the header (logo/title) of the status page should *clear* the filters, because you can already toggle them on/off by clicking the filter-button. But either way would be an improvement on the current behavior.

To continue your previous comparison, if you go to [help.github.com](https://help.github.com/) and type in the search field, the website address changes (`?query=example`) and the search results are displayed. Clicking the logo in the top-left corner resets both the page and website address.

File: index.html (hunk 1 above)
Comment (uxitten) [reply index.html:34]: Currently, When clicking on logo/title, query string in URL has been removing but there are some filter still

Comment (krilnon) [reply index.html:34]: Hmm, fair point. Let’s just go with your original approach, then.

**Comment by krilnon** [comment]
Alright, this seems like the way to go.

**Pull request merged**